### ISSUE TITLE
UDSelect onChange Not Working (Browser Error)

### ISSUE THREAD

**Issue by BryanDM** [ironmansoftware/universal-dashboard#338]
When using a UDSelect with on onChange, I receive this error in chrome when changing options. None of the script in the onChange seem to be executing.

```
HubConnection.js:167 Uncaught (in promise) Error: An unexpected error occurred invoking 'clientEvent' on the server.
    at t.callbacks.(anonymous function) (https://localhost:10006/main.18562bb9db959fec74ef.bundle.js:48:419329)
    at e.processIncomingData (HubConnection.js:259)
    at e.connection.onreceive (HubConnection.js:21)
    at WebSocket.l.onmessage (WebSocketTransport.js:50)
t.callbacks.(anonymous function) @ HubConnection.js:167
e.processIncomingData @ HubConnection.js:259
connection.onreceive @ HubConnection.js:21
l.onmessage @ WebSocketTransport.js:50

```

**Comment by adamdriscoll** [comment]
@BryanDM Can you please provide a repro? I tried this and UDSelect is working: 

```powershell
  New-UDSelect -Label "Test" -Option {
        New-UDSelectOption -Nam "Test 1" -Value "1"
        New-UDSelectOption -Nam "Test 2" -Value "2"
        New-UDSelectOption -Nam "Test 3" -Value "3"
    } -OnChange {
        Show-UDToast -Message $EventData -Title "EventData" -Position "topRight"
    }
```

**Comment by BryanDM** [comment]
@adamdriscoll 

I copy and pasted your code exactly into the following dashboard.

```
$testPage = New-UDPage -Name "test" -Content {
    
    New-UDButton -Text "Button" -OnClick {
        Show-UDToast -Message "This one Works" -Position topRight -Title ""
    }

    New-UDSelect -Label "Test" -Option {
        New-UDSelectOption -Nam "Test 1" -Value "1"
        New-UDSelectOption -Nam "Test 2" -Value "2"
        New-UDSelectOption -Nam "Test 3" -Value "3"
    } -OnChange {
        Show-UDToast -Message $EventData -Title "EventData" -Position "topRight"
    }
}

$dashboard = New-UDDashboard -Title "test" -Pages @($testPage)

Start-UDDashboard -Dashboard $dashboard -Port 10010
```
On click for buttons is working without issue, but i'm still having errors with selects. I'm attaching an image of my chrome console. The first error is when I load the page and the second is when changing the select. I'm experiencing this on both IIS and through the console of VSCode.

![image](https://user-images.githubusercontent.com/36564688/45063841-349ef280-b07e-11e8-9cff-6937bbe5cbb9.png)

**Comment by adamdriscoll** [comment]
@BryanDM Ok. Thanks. When you get a chance can you try again with 2.0.1 (just published to the gallery)? I think the should resolve the setSessionid issue. 

As for the second error, let me try and repro and get back to you.

**Comment by BryanDM** [comment]
@adamdriscoll The screenshot I just sent was running 2.0.1, same issue with 2.0

No issues on 1.7

**Comment by adamdriscoll** [comment]
Ok. Thanks for the heads up.

**Comment by gerane** [comment]
@adamdriscoll I am seeing this issue 2.0.1.

**Comment by NotBR** [comment]
Same for me also on 2.0.1 Community.

![image](https://user-images.githubusercontent.com/18089345/46907427-1bedec00-cf0a-11e8-804c-a97ecdcdba96.png)

**Comment by BryanDM** [comment]
Any chance we can get this working for 2.0.2? It's keeping me from upgrading most of my dashboards. 

Thanks!

**Comment by adamdriscoll** [comment]
Fixed in 2.1.0. PR in the pipeline. Looking to release 2.1.0 in the next couple days.
